site/content/usage/06-vpc-networking.md

```diff
@@ -160,3 +160,40 @@ See the complete example [here](https://github.com/weaveworks/eksctl/blob/master
 
 **Note**: Specifying the NAT Gateway is only supported during cluster creation and it is not touched during a cluster
 upgrade. There are plans to support changing between different modes on cluster update in the future.
+
+### Managing Access to the Kubernetes API Server Endpoints
+
+The default creation of an EKS cluster exposes the Kubernetes API server publicly but not directly from within the
+VPC subnets (public=true, private=false). Traffic destined for the API server from within the VPC must first exit the
+VPC networks (but not Amazon's network) and then re-enter to reach the API server.
+
+The Kubernetes API server endpoint access for a cluster can be cofigured for public and private access when creating
+the cluster using the cluster config file. Example below:
+
+```yaml
+vpc:
+  clusterEndpoints:
+    publicAccess:  <true|false>
+    privateAccess: <true|false>
+```
+
+There are some additional caveats when configuring Kubernetes API endpoint access:
+
+1. EKS doesn't allow one to create or update a cluster without at least one of private or public access being
+   enabled.
+1. EKS does allow creating a configuration which allows only private access to be enabled, but eksctl doesn't
+   support it during cluster creation as it prevents eksctl from being able to join the worker nodes to the cluster.
+1. To create private-only Kubernetes API endpoint access, one must first create the cluster *with* public Kubernetes API
+   endpoint access, and then use `/eksctl utils update-cluster-endpoints` to change it after the cluster is finished
+   creating.
+1. Updating a a cluster to have private only Kubernetes API endpoint access means that Kubernetes commands
+   (e.g. `kubectl`) as well as `eksctl delete cluster`, `eksctl utils write-kubeconfig`, and possibly the command
+   `eksctl utils update-kube-proxy` must be run within the cluster VPC.  This requires some changes to various AWS
+   resources.  See:
+   https://docs.aws.amazon.com/en_pv/eks/latest/userguide/cluster-endpoint#private-access
```

Comment:
I'd change this to `(see [EKS user guide](ttps://docs.aws.amazon.com/en_pv/eks/latest/userguide/cluster-endpoint#private-access))` at the end of the previous sentence.

Reply:
Done